File: linear_search/python/linear_search.py
def search(arr, x): 
    for i in range(len(arr)): 
        if arr[i] == x: 
            return i 
    return -1
arr=[10,20,3,4]
x=20
print(arr)
if(search(arr,x)==1):
    print("Element found")
else:
    print("not found")

File: linear_search/cpp/linear_search.cpp
#include<iostream>
using namespace std;
int main()
{
	int arr[100];
  int i=0;
  int n=0;
  int flag=0;
  int size=0;
  int p=0;
	cout<<"Enter size of array  "<<endl;
	cin>>size;
	cout<<"Enter Elements of the array "<<endl;
	for(i=0; i<size; i++)
	{
		cin>>arr[i];
	}
	cout<<"Enter the number to be found  ";
	cin>>n;
	for(i=0; i<size; i++)
	{
		if(arr[i]==n)
		{
			flag=1;
			p=i;
			break;
		}
	}
	if(flag==0)
	{
		cout<<"Number is not in the array";
	}
  if (flag==1)
	{
		cout<<"number is  found at array index "<<p;
	}

  return 0;
}
